docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -857,7 +857,7 @@ Administrators can increase this if they experience OAuth login state mismatch e
 
 ### oauth_skip_org_role_update_sync
 
-> **Note**: This option will soon be a legacy option in favor of OAuth provider specific `skip_org_role_sync` settings. The following sections explain settings for each provider.
+> **Note**: This option is deprecated in favor of OAuth provider specific `skip_org_role_sync` settings. The following sections explain settings for each provider.
```

Comment:
@eleijonmarck If this option is deprecated, shouldn't that be mentioned in the What's new doc and then remove the associated docs?

Reply:
yea, if we feel like this is good to go we can add it to the what's new. I am not sure as to how long we can update that before it will be released.

I think we keep the documentation with the deprecated settings to point to the new settings as long as we have this option. Once we remove the setting we can remove it from the docs